go-controller/pkg/allocator/pod/pod_annotation.go

```diff
@@ -265,6 +265,10 @@ func allocatePodAnnotationWithRollback(
 		IPs:      podAnnotation.IPs,
 		MAC:      podAnnotation.MAC,
 		TunnelID: podAnnotation.TunnelID,
+		Primary:  false,
```

Comment:
is this path expected to be hit during default net info path?

Reply:
spoke to Jaime (tl;dr currently we don't call this path from default netinfo but in future we could; so better to have that contingency in place)